typings.d.ts

```diff
@@ -16,6 +16,11 @@ declare module '@electron-elements/send-feedback' {
 
 declare module 'node-mac-notifier';
 
+declare module '@yaireo/tagify' {
+	const tagify: any;
```

Comment:
Avoid `any`.

Reply:
Can we only declare the module? Thought it hinders Intellisense, but we only need one method - `tagify.addTags()`